app/src/controllers/reservation/repositories/ShopRepository.ts

```diff
@@ -78,6 +78,19 @@ const ShopRepository: ReservationServiceSocket = {
     })).map(s => s.id)
   },
 
+  async fetchShopSchedule(shopId) {
+    const schedule = await prisma.shop.findUnique({
```

Comment:
what returns is a shop, better change it to
```suggestion
  const shop = await prisma.shop.findUnique({
```